.github/workflows/release.yaml

```diff
@@ -5,7 +5,9 @@ env:
   VSPHERE_USERNAME: ${{ secrets.VSPHERE_NKP_SRE_USERNAME }}
   VSPHERE_PASSWORD: ${{ secrets.VSPHERE_SRE_PASSWORD }}
   VSPHERE_SERVER: ${{ secrets.VSPHERE_NKP_SERVER }}
-  GOVC_URL: ${{ secrets.VSPHERE_NKP_SRE_USERNAME }}:${{ secrets.VSPHERE_SRE_PASSWORD }}@${{ secrets.VSPHERE_NKP_SERVER }}
+  GOVC_USERNAME: ${{ secrets.VSPHERE_NKP_SRE_USERNAME }}
```

Comment:
Some characters in username and password breaks GOVC_URL. So I split them as per instructions at https://github.com/vmware/govmomi/blob/main/govc/README.md#usage
```
If username or password includes special characters like \, # or : you can use GOVC_USERNAME and GOVC_PASSWORD to have a simple GOVC_URL
```